### PR TITLE
finalize sms strings

### LINKED ISSUE #3643: Verify all mobile strings before we push to l10n branch
@devinreams making a placeholder for myself to review/update all of the SMS flow strings before we do a push to the l10n branch

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #3643

### CODE REVIEW

**Comment by johngruen** [comment]
@flodolo this only deletes strings that haven't been sent to l10n yet

**Review by flodolo — APPROVED** [review]
Thanks. In general, go ahead without my review when removing strings.

**Comment by johngruen** [comment]
kk, looks like i broke a test anyway

File: frontend/src/app/components/FeaturedExperiment/index.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ export default class FeaturedExperiment extends React.Component {
     const { experiment, enabled, installed, hasAddon, sendToGA,
       eventCategory } = this.props;
     const { description, title, subtitle, slug, video_url, error } = experiment;
+    const { platforms } = experiment;
```
Comment (pdehaan) [inline frontend/src/app/components/FeaturedExperiment/index.js]: Why not destructure this on L48 with the other params?

File: frontend/src/app/components/MobileDialog/index.js [hunk 2]
```diff
@@ -82,12 +82,23 @@ export default class MobileDialog extends React.Component {
 
     const headerImg = ios_url ? (<a href={ios_url} target="_blank"><img className="mobile-header-img" src="/static/images/ios-light.svg"/></a>) : (<a href={android_url} target="_blank"><img className="mobile-header-img" src="/static/images/google-play.png"/></a>);
 
+    const learnMoreLink = "https://www.mozilla.org/en-US/privacy/websites/#campaigns";
```
Comment (pdehaan) [inline frontend/src/app/components/MobileDialog/index.js]: nit: drop the "/en-US/" in the URL and let mozilla.org serve the "best" user locale.

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/2594b2c70d46af1f24194e0555e7c1ad7117bb1a/

File: frontend/src/app/components/MobileDialog/index.js [hunk 3]
```diff
@@ -82,12 +82,23 @@ export default class MobileDialog extends React.Component {
 
     const headerImg = ios_url ? (<a href={ios_url} target="_blank"><img className="mobile-header-img" src="/static/images/ios-light.svg"/></a>) : (<a href={android_url} target="_blank"><img className="mobile-header-img" src="/static/images/google-play.png"/></a>);
```
Comment (pdehaan) [inline frontend/src/app/components/MobileDialog/index.js]: Bonus nit: Add `rel="noopener noreferrer"` to both the `target="_blank"` links.

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/826a68871484f1b6f4ed7c36476aa51b0d4b5b3e/

File: frontend/src/app/components/FeaturedExperiment/index.js [hunk 4]
```diff
@@ -45,7 +45,8 @@ export default class FeaturedExperiment extends React.Component {
   render() {
     const { experiment, enabled, installed, hasAddon, sendToGA,
       eventCategory } = this.props;
-    const { description, title, subtitle, slug, video_url, error } = experiment;
+    const { description, title, subtitle, slug, video_url, error, platforms } = experiment;
+    const isMobile = platforms.includes("ios") || platforms.includes("android");
```
Comment (meandavejustice) [inline frontend/src/app/components/FeaturedExperiment/index.js]: nit: we use `isMobile` elsewhere to determine whether or not the user is on a mobile device based on useragent. I would slightly rename this to not cause confusion.

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/e70d0182eedf07c81fdbb12057e30a8244ced61d/

File: frontend/src/app/components/FeaturedExperiment/index.js (hunk 4 above)
Comment (johngruen) [reply frontend/src/app/components/FeaturedExperiment/index.js]: cool

File: frontend/src/app/components/MobileDialog/index.js (hunk 3 above)
Comment (johngruen) [reply frontend/src/app/components/MobileDialog/index.js]: Man, this ain't even in my PR

Comment (meandavejustice) [reply frontend/src/app/components/MobileDialog/index.js]: I'll fix this in my metrics modal pr, since I have to fix conflicts there anyways

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/7fc0e8964434dea0b1653cb42cd8ea84c36f0363/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/65d54cda14039ed0cb150b4928a27c5ed2d0f5de/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3659/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/7e5c922293a25273fd44dbe12cd2f5c9294fe5d9/

**Pull request merged**